paper-input-error.html

```diff
@@ -43,6 +43,8 @@
         @apply(--paper-font-caption);
         @apply(--paper-input-error);
         position: absolute;
+        left:0;
+        right:0;
```

Comment:
I'm an idiot -- I was running off the right branch, you actually fixed this a bit, and it's now not flying out of the container.